packages/sitecore-jss-nextjs/src/utils/utils.ts

```diff
@@ -9,7 +9,8 @@ import { isEditorActive, resetEditorChromes } from '@sitecore-jss/sitecore-jss/u
  * preview deployment has unique url, we don't know exact url.
  */
 export const getPublicUrl = (): string => {
-  if (process.env.VERCEL_URL) return `https://${process.env.VERCEL_URL}`;
+  if (process.env.URL) return process.env.URL; // Netlify public URL
```

Comment:
Did you confirm `URL` works for preview deployments as well (i.e. this will be the unique preview deployment URL in this case)?

Reply:
Yes, both work without visible issues.